rubrik_cdm/cluster.py

```diff
@@ -606,3 +608,58 @@ def read_only_authorization(self, username, timeout=15):
 
         self.log("read_only_authorization: Granting read-only privilages to user '{}'.".format(username))
         return self.post("internal", "/authorization/role/read_only_admin", config, timeout)
+
+    def cluster_node_id(self):
+        """ Returns a list of node ids from all the nodes in the cluster.
+
+                Arguments:
+                    None
+
+                Keyword Arguments:
+                    None
+
+                Returns:
+                    dict -- The full API response from `POST /internal/authorization/role/read_only_admin`.
+                """
+        self.log("cluster_node_id - Getting all the node ids from the from the cluster")
+        node_id = self.get('internal', '/node')
+        return node_id
+
+    def cluster_support_tunnel(self,action):
+        """ Function that can check the status, enable or disable the support tunnel.
+
+                        Arguments:
+                            Action - with state of (Status,Enable,Disable)
```

Comment:
For the action state, let's have the option be `status, enable, disable` (i.e all lowercase) to make some of the other functions. Also, we need to add a logic check to make sure the user put in a support option. Something like

if action.lower() not in["status", "disable", "enable"] through an error (should be the RubrikInvalidParamater error -- should be an example in some of the other functions).

I also added a `.lower` call to the check to automatically lowercase the action variable provided. This just makes sure we don't run into any issues with upppercase vs lowercase etc. Need to make sure we use this throughout the code.